.werft/platform-delete-preview-environments-cron.ts

```diff
@@ -9,6 +9,8 @@ import { CORE_DEV_KUBECONFIG_PATH } from './jobs/build/const';
 // Will be set once tracing has been initialized
 let werft: Werft
 
+let previewsToDelete = [];
```

Comment:
I'd prefer not to have a global variable like this. I've commented further down on how to get rid of it ☺️

Reply:
I am also not in favor but did it because of minimal effort.